ExtensionFrSexEst.owl

```diff
@@ -254,6 +290,73 @@
     
 
 
+    <!-- http://w3id.org/rdfbones/core#Completeness2States -->
+
+    <owl:Class rdf:about="http://w3id.org/rdfbones/core#Completeness2States">
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/OBI_0001938"/>
+                <owl:someValuesFrom rdf:resource="http://www.semanticweb.org/martin/ontologies/2016/8/untitled-ontology-26#ValueSpecification.FrSexEst"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
```

Comment:
This restriction would apply to all contexts in which this class is used and is, therefore, an inadmissible extension of a core ontology concept.